server/helpers/swagger.yaml

```diff
@@ -242,6 +242,16 @@ paths:
             summary: Handle POST request for Sharing content
             tags:
                 - ShareContent
+    /api/events/events-streamer:
+        get:
+            description: Fetches events from Remote Provider and streams it to the UI
+            operationId: idGetEventStreamer
+            responses:
+                "200":
+                    description: ""
```

Comment:
@Revolyssup what do we have for a description here?

Reply:
There aren't any descriptions among the other endpoints, but there is a general description anyway on the handler itself. If we need to add a description, this should be the same for the other endpoints too

Reply:
@acald-creator you're so very much right: all of the endpoints need additional documentation. As an extensible platform, Meshery needs very well-documented APIs.  Yes, please.